src/router.ts

```diff
@@ -0,0 +1,26 @@
+import { Router } from 'express';
+import { InvalidArgument } from './utils/errors/client.error';
+import UserController from './user/user.controller';
+import { wrapAsync } from './utils/wrappers';
+
+const appRouter = Router();
+
+appRouter.get('/isAlive', (req, res) => {
+  res.status(200).send('alive');
+});
+
+appRouter.get('/api/users/:id', wrapAsync(async (req, res) => {
```

Comment:
I would recommend splitting into UserRouter.
Instead of wrapAsync you can check wrapController in utils/express.

Reply:
I see no point in splitting the router, because there is nothing else that would remain in `router.ts`.
Do you prefer I will just move the file to `users/user.router`?

Reply:
About the `wrapAsync`:
It's a function that wraps an asynchronous express request handler function with an error handler. Later it will also contain also contain logs.

I think it supply the need better than `wrapController` in the template. Especially, in this light service.